app/src/main/java/org/oppia/app/home/RouteToRecentlyPlayedListener.kt

```diff
@@ -1,6 +1,6 @@
 package org.oppia.app.home
 
 /** Listener for when an activity should route to [ContinuePlayingActivity]. */
```

Comment:
Missed out this comment.

Reply:
Sorry, really didn’t saw this one, thanks 👍

Reply:
Have made this change now